### ISSUE TITLE
PS Eye on Ubuntu 12.04 doesn't pick up controller

### ISSUE THREAD

**Issue by rhololkeolke** [thp/psmoveapi#64]
I've already emailed the mailing list about this, but I figured I'd also file an issue here. I've been trying to get the PS Eye to detect the PSMove controllers using the Tracker methods. I can successfully open the PS Eye camera and get images from it.

When I start the test_tracker program the window showing the image stream with the tracking overlay appears but it fails to actually track the controller. Instead it seems to move away from the controller or stick in the upper left corner of the image. However, the test_tracker program works perfectly on my macbook using the iSight camera.

I have the PSEye set to the blue dot. I've also tried it with the red dot but it doesn't seem to matter.

I did notice that the program runs slightly differently between my mac and Ubuntu. On my mac it asks me to cover the iSight with the controller and press a button then move back and wait. It does not ask me to do that on Ubuntu. Also on Ubuntu it complains about not having a lens calibration file. I did not see any documentation on calibrating a lens.

**Comment by thp** [comment]
Closing this for now, although we should probably remove the color mapping code if it causes that many problems. For this, someone should look into how calibration currently works with the PS Eye. There have been some tracker improvements in Git lately.

**Comment by EibrielInv** [comment]
I'm having the same issue with version 4.0.8 on Ubuntu 18.04 and a PS3 camera.
Building tracker config:
```
Tracker
    Tracker library:  Yes (OpenCV version 3.2.0)
    PS Eye support:   Yes
    Use PS3EYEDriver: No (disabled)
```

The tracker is unable to find the light about 80% of the time, it locks on random places (a wall, my face).

I'm in a low light environment so I tried dimming down the controller's light, and the camera exposition with no effect at all.

I also tried the hack on https://github.com/thp/psmoveapi/issues/64#issuecomment-17060868 with no effect.

I used the exact same setup ages ago on version 3.0.0 with no issues.
